test/index.test.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-undef */
```

Comment:
pq precisou remover essa regra?

Reply:
honestamente, não me lembro o motivo desse erro estar acontecendo, mas acredito que consegui resolver e não precisei mais dela.

Reply:
me lembrei, eu acho! antes, por algum motivo, os métodos do mocha, como describe e it, davam como não definidos, então precisei remover a regra. porém, após uma mudança no tsconfig, acredito eu que adicionar "mocha" a "types", não foi mais necessário essa regra.